api/v1alpha1/etcdpeer_types.go

```diff
@@ -47,11 +47,11 @@ type EtcdPeerSpec struct {
 	// pods. As per the etcd documentation, etcd will ignore bootstrap
 	// instructions if it already knows where it's peers are.
 	// +optional
-	Bootstrap *Bootstrap `json:"bootstrap,omitempty"`
+	Bootstrap Bootstrap `json:"bootstrap,omitempty"`
 
 	// Storage is the configuration of the disks and mount points of the Etcd
 	// pod.
-	Storage *EtcdPeerStorage `json:"storage,omitempty"`
+	Storage EtcdPeerStorage `json:"storage,omitempty"`
```

Comment:
Is the `storage` field required? If so, :+1: (and remove omitempty as it has no effect)

Reply:
It is required. I've remove omitempty.